maps_dashboards/public/application.tsx

```diff
@@ -10,16 +10,18 @@ import { AppPluginStartDependencies } from './types';
 import { MapsDashboardsApp } from './components/app';
 
 export const renderApp = (
-  { notifications, http }: CoreStart,
+  { notifications, http, savedObjects }: CoreStart,
   { navigation }: AppPluginStartDependencies,
-  { appBasePath, element }: AppMountParameters
+  { appBasePath, element, setHeaderActionMenu }: AppMountParameters
 ) => {
   ReactDOM.render(
     <MapsDashboardsApp
       basename={appBasePath}
       notifications={notifications}
       http={http}
       navigation={navigation}
+      savedObjectsClient={savedObjects.client}
+      setHeaderActionMenu={setHeaderActionMenu}
```

Comment:
All of this can ideally be passed down as a single services object e.g. https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_builder/public/plugin.ts#L84-L97

Reply:
Yes, updated, all components changed to use a single service object.